### PR TITLE
Feat data manipulation

### PR DESCRIPTION
**Pull request by Fav8**

### CODE REVIEW

File: tests/controllers/ManipulateDataController.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import {describe, test} from '@jest/globals';
+import {containerBuilder} from '../testContainer';
+import ManipulateDataController from '../../src/controllers/ManipulateDataController';
+import {Request, Response} from 'express';
+import * as mockResponse from '../mockResponses/mockResponse1.json';
+import fn = jest.fn;
+
+describe('Test Controller', () => {
```
Comment (tinazhouhui) [inline tests/controllers/ManipulateDataController.ts:8]: would probably change the name of the test :)

File: src/controllers/ManipulateDataController.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import {Request, Response, Router} from 'express';
+import {AmountOfPeople} from '../models/helpers';
+import BaseController from './BaseController';
+
+export default class ManipulateDataController extends BaseController {
+  register (router: Router): void {
+    router.route('/manipulateData').post((request: Request, response: Response) => {
+      return this.actionDefault(request, response);
+    });
+  }
```
Comment (tinazhouhui) [inline src/controllers/ManipulateDataController.ts:10]: @Fav8 why do we need this endpoint? 🤔 remind me pls?

Comment (tinazhouhui) [reply src/controllers/ManipulateDataController.ts:10]: oh noo this is the old one, gotcha

**Review by tinazhouhui — CHANGES_REQUESTED** [review]
@Fav8 just to make sure I understand correctly, this is the previous version of the data manipulation right?cause the helper functions should all be in our Sentiment Model and there should not really be an endpoint for /manipulateData?

**Comment by Fav8** [comment]
closing, outdated

**Pull request closed without merging**